src/osuTK.Android/GameViewBase.cs

```diff
@@ -1099,12 +1099,8 @@ public Point PointToScreen (Point point)
         ///   </para>
         /// </remarks>
         public Rectangle ClientRectangle {
-            get {
-                throw new NotSupportedException ();
-            }
-            set {
-                throw new NotSupportedException ();
-            }
+            get => new Rectangle(0, 0, Width, Height);
+            set { }
```

Comment:
```suggestion
            set {	
                throw new NotSupportedException ();	
            }
```

We should leave the setter throwing exceptions if we can (please check this still works).